examples/flutter_gallery/lib/demo/cupertino/cupertino_alert_demo.dart

```diff
@@ -15,181 +14,190 @@ class CupertinoAlertDemo extends StatefulWidget {
-                ),
-              );
-            },
-          ),
-          const Padding(padding: EdgeInsets.all(8.0)),
-          CupertinoButton(
-            child: const Text('Alert with Buttons'),
-            color: CupertinoColors.activeBlue,
-            padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
-            onPressed: () {
-              showDemoDialog<String>(
-                context: context,
-                child: const CupertinoDessertDialog(
-                  title: Text('Select Favorite Dessert'),
-                  content: Text('Please select your favorite type of dessert from the '
-                      'list below. Your selection will be used to customize the suggested '
-                      'list of eateries in your area.'),
-                ),
-              );
-            },
-          ),
-          const Padding(padding: EdgeInsets.all(8.0)),
-          CupertinoButton(
-            child: const Text('Alert Buttons Only'),
-            color: CupertinoColors.activeBlue,
-            padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
-            onPressed: () {
-              showDemoDialog<String>(
-                context: context,
-                child: const CupertinoDessertDialog(),
-              );
-            },
-          ),
-          const Padding(padding: EdgeInsets.all(8.0)),
-          CupertinoButton(
-            child: const Text('Action Sheet'),
-            color: CupertinoColors.activeBlue,
-            padding: const EdgeInsets.symmetric(vertical: 16.0, horizontal: 36.0),
-            onPressed: () {
-              showDemoActionSheet<String>(
-                context: context,
-                child: CupertinoActionSheet(
-                  title: const Text('Favorite Dessert'),
-                  message: const Text('Please select the best dessert from the options below.'),
-                  actions: <Widget>[
-                    CupertinoActionSheetAction(
-                      child: const Text('Profiteroles'),
-                      onPressed: () {
-                        Navigator.pop(context, 'Profiteroles');
-                      },
-                    ),
-                    CupertinoActionSheetAction(
-                      child: const Text('Cannolis'),
-                      onPressed: () {
-                        Navigator.pop(context, 'Cannolis');
-                      },
-                    ),
-                    CupertinoActionSheetAction(
-                      child: const Text('Trifle'),
-                      onPressed: () {
-                        Navigator.pop(context, 'Trifle');
-                      },
-                    ),
-                  ],
-                  cancelButton: CupertinoActionSheetAction(
-                    child: const Text('Cancel'),
-                    isDefaultAction: true,
+      child: DefaultTextStyle(
+        style: CupertinoTheme.of(context).textTheme.textStyle,
```

Comment:
Shouldn't the CupertinoTheme's text style already be the default text style at this point?

Reply:
Material Theme and CupertinoTheme don't actually apply any text styles. The app does. And the Material app above only applied the default Material theme's font styles.